practical_09/band.py

```diff
@@ -0,0 +1,29 @@
+"""
+Band class
+"""
+
+
+class Band:
+    """Band class"""
+
+    def __init__(self, name=""):
+        """Construct a Band with a name and empty musician collection."""
```

Comment:
As Lindsay might say, this sounds a bit "solution domain". Does someone using your class need to know you hold an empty musician collection. In mine I used the class docstring ("""Band class""") to say, "Describes a band, which can have musicians", for example.